### PR TITLE
Clarify SSO permissions mapping docs

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: docs/advanced-features/sso-permissions.md [hunk 1]
```diff
@@ -4,27 +4,29 @@
...
-> Note: Roles used by configuration can't be removed or renamed.
-
-> Note: The user who sets the configuration will never have their admin flag revoked.
-
-> Note: After configuration is set, any user who logs in via SSO can't be manually
-assigned roles or admin permissions.
+> Note: You cannot delete or rename roles referenced by the SSO configuration.
+> Note: The last user to set a the mapping configuration becomes an admin in
+> perpetuity.
+> Note: Once a configuration is set you can no longer manually assign roles to
```
Comment (sir-sigurd) [inline docs/advanced-features/sso-permissions.md:19]:
```suggestion
> Note: You cannot delete or rename roles referenced by the SSO configuration.

> Note: The last user to set a the mapping configuration becomes an admin in
> perpetuity.

> Note: Once a configuration is set you can no longer manually assign roles to
```

**Review by sir-sigurd — APPROVED** [review]

File: docs/advanced-features/sso-permissions.md [hunk 2]
```diff
@@ -4,27 +4,29 @@
 
 ## Overview
 
-This feature allows Quilt admin to configure what roles and admin flag value are
-assigned to the user who logs in via SSO based on the user's ID token claims.
+_SSO permissions mapping_ enables administrators to specify which roles are assigned
```
Comment (sir-sigurd) [inline docs/advanced-features/sso-permissions.md:7]: I used wording "Quilt admin" to make it clear that it's not AWS admin
(can be ignored)

Comment (akarve) [reply docs/advanced-features/sso-permissions.md:7]: Your wording is better but I don't want to trigger a re-review.

**Comment by drernie** [comment]
Closing old @akarve PRs

**Pull request closed without merging**